Samples.xcodeproj/project.pbxproj

```diff
@@ -825,9 +833,11 @@
 				00B04FB3283EEB830026C882 /* Display overview map */,
 				1C56B5DE2A82C02D000381DA /* Display points using clustering feature reduction */,
 				E004A6D528465C70002A1FE6 /* Display scene */,
+				D742E48E2B04132B00690098 /* Display web scene from portal item */,
 				E070A0A1286F3B3400F2B606 /* Download preplanned map area */,
 				E004A6EE284E4B7A002A1FE6 /* Download vector tiles to local cache */,
 				1C26ED122A859525009B7721 /* Filter features in scene */,
+				D73FCFF32B02A3AA0006360D /* Find address with reverse geocode */,
```

Comment:
It seems some commits from the other PR slipped in here. Not a big deal, you can reorganize these commits once the other PR is merged.